net-vpn/protonvpn-cli/protonvpn-cli-2.2.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
+
+inherit distutils-r1
+
+DESCRIPTION="A VPN command-line tool from protonvpn - python rewrite"
+HOMEPAGE="https://protonvpn.com https://github.com/ProtonVPN/protonvpn-cli-ng"
+SRC_URI="https://github.com/ProtonVPN/${PN}-ng/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+KEYWORDS="~amd64"
+SLOT="0"
+
+RDEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
```

Comment:
You should to define this via eclass, 
```
 * DISTUTILS_USE_SETUPTOOLS value is probably incorrect
 *   value:    DISTUTILS_USE_SETUPTOOLS=bdepend (default?)
 *   expected: DISTUTILS_USE_SETUPTOOLS=rdepend
```
Check the devmanual page for how to use `DISTUTILS_USE_SETUPTOOLS`

Could you also squash your commits into a single one? I think the merge commit in the middle somehow messes this up, it doesn't apply cleanly at all.

Reply:
I have opened a new PR with the changes applied at https://github.com/gentoo/gentoo/pull/14725